frontend/src/modules/sidebar/templates/sidebarFilter.hbs

```diff
@@ -13,7 +13,7 @@
         </div>
     </div>
     <div class="sidebar-filter-search">
-        <input type="text" placeholder="Filter Tags" class="sidebar-filter-search-input" />
+      <input type="text" placeholder="{{t 'app.filterbytags'}}" class="sidebar-filter-search-input" />
```

Comment:
Thanks for the PR! Please could you fix the indentation on this line?

Reply:
@canstudios-nicolaw thanks for pointing this out. fixed.